src/Layout/redist/targets/GenerateLayout.targets

```diff
@@ -459,16 +468,13 @@
 
     <ItemGroup>
       <AllFileOfRestoredAppHostPackage Include="$(AppHostRestorePath)\**\*.*" />
-      <NativeRestoredAppHostNETCore Include="@(AllFileOfRestoredAppHostPackage)"
-          Condition="'%(FileName)%(Extension)' == '$(AppHostExecutableName)'" />
+      <NativeRestoredAppHostNETCore Include="@(AllFileOfRestoredAppHostPackage)" Condition="'%(FileName)%(Extension)' == '$(AppHostExecutableName)'" />
     </ItemGroup>
 
     <Error Condition="@(NativeRestoredAppHostNETCore->Distinct()->Count()) != 1"
            Text="Failed to determine the $(NETCoreAppHostPackageName) executable in @(AllFileOfRestoredAppHostPackage)" />
 
-    <Copy SourceFiles="@(NativeRestoredAppHostNETCore)" DestinationFolder="$(AppHostTemplatePath)" />
-
-    <Message Text="Copy from @(NativeRestoredAppHostNETCore) to $(AppHostTemplatePath)." Importance="High" />
```

Comment:
Lost a message?

Reply:
That message isn't displayed anymore due to the terminal logger and as the copy task already emits this message with normal importance I didn't see the necessity for it anymore.